File: c#/uri/1007.cs
using System;
class URI{
    static void Main(){
        int a = Int32.Parse(Console.ReadLine().Trim());
        int b = Int32.Parse(Console.ReadLine().Trim());
        int c = Int32.Parse(Console.ReadLine().Trim());
        int d = Int32.Parse(Console.ReadLine().Trim());
        int dif = (a * b - c * d);
        Console.WriteLine("DIFERENCA = {0}", dif);
    }
}

File: js/URI/1007.js
var input = require("fs").readFileSync("URI/in", "utf-8");
var lines = input.split('\n');
var a = parseInt(lines.shift());
var b = parseInt(lines.shift());
var c = parseInt(lines.shift());
var d = parseInt(lines.shift());
var dif = (a * b - c * d);
console.log(`DIFERENCA = ${dif}`);